### ISSUE TITLE
Is Unturned 3.0 dead already?

### ISSUE THREAD

**Issue by Trojaner25** [SmartlyDressedGames/Unturned-3.x-Community#210]
![](https://cdn.discordapp.com/attachments/331560951583080449/411402010420838410/unknown.png)
(Unturned 4)

![](https://cdn.discordapp.com/attachments/331560951583080449/411402110744395778/unknown.png)
(Unturned 3) (below red line are the ones with activity from Nelson, like adding labels or commenting)

Is Unturned 3 abandoned already?
Last update was also 20 days ago too...

**Comment by N1GHTPATROL** [comment]
Yeah, Nelson is killing the game every day he waits. No new content, doesn't even fix what's already here. There are so many bugs it's not even funny. But yeah, one can make the argument 4.x is going to fix all this. However Nelson has always been horrible at communication with the community. If you're not Nelson, I wouldn't worry about reading this all.

I'd love to see 4.x fix everything wrong with 3.x and then some, but I'm honestly loosing faith. Nelson isn't taking community feedback we give him, and that is a major problem for any game developer. You look at ANY other active game- the developers are always in communication with their players. Then you look at Unturned, where Nelson has **5** messages in total on his official discord with many thousands of his players in it. That's a problem...

I'm not here to make a rant about how horribly optimized 3.x is like everyone else. I'm honestly here for what is to come. I'm not here to complain, I'm here to recommend to Nelson what I think he should do. From a server owner's perspective, the following is what I think Nelson needs to work focus on:

• Communication: All of the popular games have it. Nelson, if you want your game to be as big as it was in 2016, you're going to have to talk with your community. A lot. Give your players faith in your game, something me, @Trojaner25, and the entire community have been lacking for a long time now. We all enjoy this community GitHub you've created for us to report bugs, and I'd say that was a great addition. However when you're inactive on our only lifeline of support, there is a problem.

• Modding: In my opinion, modding is easily one of the most important aspects of a game. Let the community do the cool stuff for you, you just need to give them the ability. And no, not just the modding we have in 3.x. I'd like to see easy custom UI support like other games have. Custom shaders, custom texture-packs, custom player models, custom chat, custom.. everything you can make custom. We want to do what you do. The more modular your game is, the more different and attractive your community will make it.

• Optimization: Also extremely important and a rare sight in 3.x. So far all versions of Unturned haven't been well optimized. Rubber banding, 50 ping on local servers, player models jumping around, this all needs to go. In my opinion, I believe lag is the number one reason people leave your game. I shouldn't be getting a higher frame rate in GTA V Redux Mod max shaders than on your game. If you want your game to be on the top, it doesn't need to look amazing, you just need to have it perform good for what you make.

• Quality: I think the community as a whole would like to see some improvements in the general quality of Unturned. I'm not talking about high shaders etc. I'm talking about your hand being a square. However I understand you like to keep the general look of players the same, and that's fine. But for clients at least, I think a hand of some sort would be great for modders. Guns floating in the middle of a block isn't any sort of appealing.

• Servers: If it weren't for community hosted servers, I'd say Unturned would be just about dead. So thanks for that. However, I think we would all agree that more server end controls are going to help the community as a whole. Dedicated (instead of clinet) servers for Unturned are going to be a bare minimum. Server loading mods (just like 3.x) in 4.x would be amazing, also other mods mentioned above would be great if the server could download them. My dream for Unturned is for players to be able to join each server and get a completely unique experience. Something Unturned definitely has the potential for.

Good luck with everything Nelson, hopefully I've helped you even in the slightest way.

- NightPatrol

**TL;DR**: I hopefully gave Nelson some insight into what we want as a community.

**Comment by SDGNelson** [comment]
I know it's a little hidden away, but I've got a blog where I've been making posts about 4.x developments - in the most recent one I mentioned that I was going to focus on 4 for the next week or two:
https://blog.smartlydressedgames.com/unturned-4-x-devlog-004/

I'm a member of a lot of community discords, but I have been considering setting up an official one. You're right though: communication is one of my weakest points as a developer.

Feedback wise I don't think that's fair - I take community feedback into consideration for every decision.

Regarding all of your other points I think you can rest assured that they're all thoroughly covered in 4! :)

**Comment by N1GHTPATROL** [comment]
Good to hear. Is there a forum/board we can post ideas for 4.x as a community, or at least, will there be in the future?

**Comment by Trojaner25** [comment]
https://forum.smartlydressedgames.com/c/unturned-4
